Sources/Customization/DefaultEventDispatcher.swift

```diff
@@ -256,12 +254,12 @@ open class DefaultEventDispatcher: BackgroundingCallbacks, OPTEventDispatcher {
 extension DefaultEventDispatcher {
     
     func addProjectChangeNotificationObservers() {
-        observerProjectId = NotificationCenter.default.addObserver(forName: .didReceiveOptimizelyProjectIdChange, object: nil, queue: nil) { [weak self] (notif) in
+        observerProjectId = NotificationCenter.default.addObserver(forName: .didReceiveOptimizelyProjectIdChange, object: nil, queue: nil) { [weak self] (_) in
```

Comment:
This would never happen since the project config is based on the client/ sdk key.

Reply:
A good catch. I see it's never called in the current implementation of ProjectConfig. We'll keep this as is and we'll see if we'll have the usage case later.